src/pod.jl

```diff
@@ -241,3 +241,19 @@ function isk8s()
     end
     return in_kubepod
 end
+
+
+"""
+    az_node_selector!() -> Dict
+
+Add a nodeSelector to a new pod that will ensure that it runs in the same availability zone
+as the manager pod.
+"""
+function az_node_selector!(pod)
+    pod_name = ENV["HOSTNAME"]
```

Comment:
Should use the name from `pod` and not rely on the `HOSTNAME`

Reply:
Isn't `pod` a manifest of the pod to be created? It doesn't contain information about the current pod, does it?